pkg/demo/demo.go

```diff
@@ -100,6 +87,75 @@ AND (
 	)
 	fmt.Printf("sql: %s\n", sql)
 
+	return app.collectCells(ctx, sql)
+}
+
+// bboxQuery returns the csv table for LSOAs intersecting with the given bbox
+//
+func (app *Demo) bboxQuery(ctx context.Context, bbox string, cats []string) (string, error) {
+	var p1lat, p1lon, p2lat, p2lon float64
+	fields, err := fmt.Sscanf(bbox, "%f,%f,%f,%f", &p1lat, &p1lon, &p2lat, &p2lon)
+	if err != nil {
+		return "", fmt.Errorf("scanning bbox %q: %w", bbox, err)
+	}
+	if fields != 4 {
+		return "", ErrMissingParams
+	}
+
+	// Construct SQL
+	//
+	template := `
+SELECT
```

Comment:
is there any handy string function that could remove common indents from this multiline string and avoid breaking the indendation here? something equivalnent to [python's textwrap.dedent](https://docs.python.org/3/library/textwrap.html#textwrap.dedent)?

Reply:
That's common practice in go. Also most have gofmt running on the code automatically when it writes a file.